<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EvtExport</class>
 <widget class="QDialog" name="EvtExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Exportation par projet</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidget" name="listWidgetProjets"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="buttonAnnulerProjet">
           <property name="text">
            <string>Annuler</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonExportProjet">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Exporter</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Exportation par semaine</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QToolButton" name="flecheGauche">
           <property name="text">
            <string>...</string>
           </property>
           <property name="arrowType">
            <enum>Qt::LeftArrow</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Semaine du :</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateLundi">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>au :</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateDimanche">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="flecheDroite">
           <property name="text">
            <string>...</string>
           </property>
           <property name="arrowType">
            <enum>Qt::RightArrow</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="buttonAnnulerSemaine">
           <property name="text">
            <string>Annuler</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonExportSemaine">
           <property name="text">
            <string>Exporter</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonAnnulerProjet</sender>
   <signal>clicked()</signal>
   <receiver>EvtExport</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAnnulerSemaine</sender>
   <signal>clicked()</signal>
   <receiver>EvtExport</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
